.github/workflows/release-upload.yml

```diff
@@ -135,7 +135,7 @@ jobs:
     strategy:
       matrix:
         # TODO generalize
-        arch: [x86_64-linux, x86_64-darwin, aarch64-darwin, aarch64-linux]
+        arch: [x86_64-linux, x86_64-darwin, aarch64-darwin, aarch64-linux, win64]
```

Comment:
Could you update the comment in line 32, please?

Reply:
Addressed in the same commit